tag_hierarchy/commands/nodes.cpp

```diff
@@ -133,6 +134,17 @@ Nodes::ProcessRequest(std::vector<NodeType> &nodes)
         }
         auto props = graph_[*iter].properties;
         props["model_ids"] = valid_model_ids;
+
+        // Write suppression property to all nodes of the graph.
+        // Is the node suppressed?
```

Comment:
This could be written as
props["issuppressed"] = (suppressed_nodes.find(*iter) != suppressed_nodes.end());
If you prefer the verbosity I'm ok with it, but I personally avoid if statements as much as possible.

Reply:
I liked your version better. Changing.